### DISCUSSION TITLE
Unable to get the transparency for the LineNr working.

### DISCUSSION THREAD

**Discussion by vivianludrick** [olimorris/onedarkpro.nvim#267] [Q&A] [answered]
Tried things:
- tried the default config with the transparency option enabled.
- tried to explicitly mention the "NONE" for LineNr.

![image](https://github.com/user-attachments/assets/f4b1f521-b942-4ec2-a5a6-d5398599ce49)

**Comment by olimorris** [comment, marked as answer]
It must be a plugin overriding it. Check `:verbose hi LineNr` and see what the value is and what's setting it.

**Reply by vivianludrick** [reply]
Thanks for this. Finally got it to work. I set the colorscheme in the config function is which is run before the opts or something like that in lazy. Hence the options weren't set correctly.
